den0bot/Bot.cs

```diff
@@ -65,6 +65,14 @@ private string GreetNewfag(string username, long userID)
 
         public void ProcessMessage(Message msg)
         {
+            if (msg == null ||
+                (msg.Type != MessageType.TextMessage &&
```

Comment:
This should be split in two by moving `(msg.Type != MessageType.TextMessage && msg.Type != MessageType.PhotoMessage) down ` because we need to pass service messages through to see if we got any new members.